README.md

```diff
@@ -9,54 +9,6 @@ For more about us, see [our page on the PSF wiki](https://wiki.python.org/psf/Pr
 This repository holds public resources for you to use:
 
 * [Available grants and similar funds](./funders.md)
-* [Budgeting and proposal writing links](./resources.md)
+* [Budgeting and proposal writing guides](./resources.md)
+* [Projects and example proposals](./projects.md)
```

Comment:
```suggestion
* [Previous projects and example proposals](./projects.md)
```

Reply:
Prospective projects are also included in this list.